vmware_aria_operations_integration_sdk/api/integration-sdk-definition-endpoint.json

```diff
@@ -296,6 +322,9 @@
         "type": "object",
         "description": "Definition of the object model returned in CollectResult",
         "properties": {
+          "schema_version":{
+            "type":"int"
```

Comment:
I thought it was strange that the generated code was using a special 'ModelInt' type for this - Now that I see what we used for `display_order` I think it's because we want `integer` for the type:
```
schema_version:
  type: integer           
  format: int32
```